src/latexify/codegen/function_codegen_test.py

```diff
@@ -40,6 +40,25 @@ def test_visit_functiondef_use_signature() -> None:
     assert FunctionCodegen(use_signature=True).visit(tree) == latex_with_flag
 
 
+def test_visit_functiondef_ignore_docstring() -> None:
```

Comment:
Add another test to check if the `Constant` in the middle of function can be skipped as well.